test/e2e/run_networking_test.go

```diff
@@ -477,6 +477,17 @@ var _ = Describe("Podman run networking", func() {
 		Expect(session.ExitCode()).To(Equal(0))
 	})
 
+	FIt("podman run --uidmap /etc/hosts contains --hostname", func() {
```

Comment:
FIt -> It

Reply:
oops.  Too early in the morning I guess.